src/instruments/src/ND/elements/messages/TcasWxrMessages.tsx

```diff
@@ -42,7 +43,7 @@ export const TcasWxrMessages: FC<{ modeIndex: Mode }> = ({ modeIndex }) => {
         leftMessage = { text: 'TA ONLY', color: 'White' };
```

Comment:
This should probably be `centerMessage` instead?

Reply:
Fixed! :)